yaksh/views.py

```diff
@@ -813,7 +816,8 @@ def download_csv(request, questionpaper_id):
     if not is_moderator(user):
         raise Http404('You are not allowed to view this page!')
     quiz = Quiz.objects.get(questionpaper=questionpaper_id)
-    if quiz.course.creator != user:
+    course = Course.objects.filter(teachers=user, id=quiz.course.id)
+    if quiz.course.creator != user and not course:
```

Comment:
Another way to do this: if user != quiz.course.creator and user not in quiz.course.teachers